src/components/NewFooter/MenuContent.tsx

```diff
@@ -27,32 +27,55 @@ const MenuContent: React.FC = () => {
         <SectionHeader>Learn</SectionHeader>
         <Column>
           {learnLinks.map((link: LinkItem) => (
-            <LearnLink key={link.cta} to={link.url}>
+            <LearnLink
+              key={link.cta}
+              to={link.url}
+              onClick={trackMenuEvent(`Learn: ${link.cta}`)}
+            >
```

Comment:
Consider including the prefix in the `trackMenuEvent` function so this component still works when we start using this component in the app menu

```tsx
function trackMenuEvent(label: string) {
  trackEvent(
    EventCategory.FOOTER,
    EventAction.NAVIGATION,
    `Learn: ${label}`
  );
}
```